### PR TITLE
First commit of augmented reality article.

### LINKED ISSUE #2089: content: AR for the Web
**Author**: @jmedley
**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]

**Target publish date:** 2020-02-13
- [x] Check this box if this is a hard deadline.

- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

Check out the [web.dev writing process](https://web.dev/handbook/quick-start/).

When you're ready, make a copy of [this template](https://docs.google.com/document/d/1lgaNIEnXZf-RB8_p9RK22QEgpXJqnu77pLWVWVy4nuw/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [x] 1. I've done my proposal <add a link to your proposal doc>
- [ ] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [ ] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request #<add PR number>
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #2089

Note: If this is written content for the site please include the issue number from your [content proposal](https://github.com/GoogleChrome/web.dev/issues?q=is%3Aissue+is%3Aopen+label%3A%22content+proposal%22).

Changes proposed in this pull request:

- Adds an article for implementing augmented reality using the WebXR Device API.

### CODE REVIEW

**Comment by jpmedley** [comment]
@kaycebasques @mfriesenhahn Can I get one of you to review this, please.

**Comment by mfriesenhahn** [comment]
I can take it! @jpmedley, when do you need it by?

**Comment by jpmedley** [comment]
ASAP. All of the WebXR articles on are blocking the beta release post. 

This got a bit away from me. I can only apologize. No. Scratch that. I can apologize and promise to be your lackey for an equivalent amount of time in payment of the debt.

File: src/site/content/en/blog/web-ar/index.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+session is largely the same. Running a frame loop is largely the same. The
+differences lie in configurations that allow content to be shown appropriately
+for augmented reality. If you're not familiar with the basic concepts of WebXR,
+you should read my earlier posts on the WebXR Device API, or at least be
+familiar with the topics covered therein. You should know how to [request and
+enter a session](https://web.dev/vr-comes-to-the-web/) and you should know how
+to run [a frame loop](https://web.dev/vr-comes-to-the-web-pt-ii).
+
+For information on hit testing, see the companion article [article
+title](article url). The code in this article is based on the Immersive AR
```
Comment (kaycebasques) [inline src/site/content/en/blog/web-ar/index.md]: Missing link

File: src/site/content/en/blog/web-ar/index.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+Session sample
+([demo](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html)
```
Comment (kaycebasques) [inline src/site/content/en/blog/web-ar/index.md]:
```suggestion
([demo](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html),
```

FYI @jpmedley if you click the **Commit Suggestion** button you can apply the commit that fixes this from the GitHub UI.

File: src/site/content/en/blog/web-ar/index.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+[source](https://github.com/immersive-web/webxr-samples/blob/master/immersive-vr-session.html))from
```
Comment (kaycebasques) [inline src/site/content/en/blog/web-ar/index.md]:
```suggestion
[source](https://github.com/immersive-web/webxr-samples/blob/master/immersive-vr-session.html)) from
```

Adds a space between `)` and `from`

File: src/site/content/en/blog/web-ar/index.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+Consider the second use case. Imagine simulating placing a life-size
+representation of a virtual object in a real scene. Once placed, the image stays
+on the selected surface, appears the size it would be if the actual item were on
+that surface, and allows the user to move around it as well as closer to it or
+farther from it. This gives viewers a deeper understanding of the object than is
+possible with a two dimensional image.
+
+I'm getting a little ahead of myself. To actually do what I've described, you
+need AR functionality and some means of detecting surfaces. This article covers
+the former. The accompanying article on the WebXR Hit Test API covers the
```
Comment (kaycebasques) [inline src/site/content/en/blog/web-ar/index.md]: Can we link to this Hit Test API article? Ignore this comment if it doesn't exist yet.

File: src/site/content/en/blog/web-ar/index.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+  if (supported) {
+    xrButton.addEventListener('click', onButtonClicked);
+    xrButton.textContent = 'Enter AR';
+    xrButton.enabled = supported; // supported is Boolean
+  }
+}
+```
+
+As before, this enables an 'Enter AR' button. When the user clicks it, call
+`xr.requestSEssion()`, also passing `'immersive-ar'`.
```
Comment (kaycebasques) [inline src/site/content/en/blog/web-ar/index.md]:
```suggestion
`xr.requestSession()`, also passing `'immersive-ar'`.
```

requestSEssion => requestSession

File: src/site/content/en/blog/web-ar/index.md (hunk 4 above)
Comment (jpmedley) [reply src/site/content/en/blog/web-ar/index.md]: It's the one with the missing link. I added a parenthetical to point that out.

**Pull request merged**